content/webapp/components/ItemRequestModal/ConfirmedDialog.tsx

```diff
@@ -22,7 +24,7 @@ const ConfirmedDialog: FunctionComponent<ConfirmedDialogProps> = ({
       </Header>
       <p>
         It will be available to pick up from the library (Rare Materials Room,
-        level&nbsp;3) for one week from your selected pickup date.
+        level&nbsp;3) for one week from {pickUpDate}.
```

Comment:
As `pickUpDate` is optional, this could be an incomplete sentence. Although IIRC, it just happens in the first milliseconds though right? Might still be nice to account for it in case something breaks?

```suggestion
   level&nbsp;3) for one week from {pickUpDate || 'your selected pickup date'}.
```

Reply:
🙌